private/bufpkg/bufmodule/module_data.go

```diff
@@ -46,15 +46,13 @@ type ModuleData interface {
 
 	// BufYAMLObjectData gets the v1beta1 or v1 buf.yaml ObjectData.
 	//
-	// This is always present, even if the Module was created from a v2 buf.yaml file. The BSR will
```

Comment:
Interesting. If the only purpose of this is to calculate the digest, why would a v1 buf.yaml be synthesized?

> This is used for digest calculations. It is not used otherwise.

Reply:
Nevermind, the answer is obvious in retrospect.